pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -143,6 +166,75 @@ func CalculateInterPodAffinityPriorityReduce(pod *v1.Pod, meta interface{}, shar
 	return nil
 }
 
+func (p *podAffinityPriorityMap) processExistingPod(existingPod *v1.Pod, existingPodNodeInfo *schedulernodeinfo.NodeInfo) error {
```

Comment:
this is just moving the processPod to a separate function instead of embedding it. This help showing it better in profiling.